CONTRIBUTING.md

```diff
@@ -1,43 +1,40 @@
-Before participating in this project's activities, please read and follow [code of conduct](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/blob/master/CODE_OF_CONDUCT.md).
+Before opening a pull request, please review and follow the guidelines outlined in this document. Additionally, be sure to read our [Code of Conduct](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/blob/master/CODE_OF_CONDUCT.md) to understand expected behavior within the project.
 
-Please create an [issue](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/issues) before submitting a pull request. If it is a proposal for a new feature or changing existing functionality, initiate a discussion with maintainers first. If it's a fix for know bugs, a discussion is not required.
+If you plan to submit a pull request, we ask that you first create an [issue](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/issues). For new features or modifications to existing functionality, please start a discussion with the maintainers. For straightforward bug fixes, an issue is enough without a preliminary discussion.
 
 ## Developing
 
-- [Install Go.](https://golang.org/doc/install)
-- The project uses Makefile. Install `make` command.
-- Fork this repository.
-- Clone the forked repository.
--  Make changes (see [Formatting](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/blob/master/CONTRIBUTING.md#formatting)) and commit to your fork. Commit messages should follow the [Conventional Commits](https://www.conventionalcommits.org/) style.
-- Add appropriate unit tests (see [Testing](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/blob/master/CONTRIBUTING.md#testing)) for your changes.
-- Update documentation if appropriate.
-- Create a pull request with your changes.
-- Github action will run the necessary checks against your pull request.
-- A maintainer will review the pull request once all checks are
-- A maintainer will merge and create a release (see [Releasing](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/blob/master/CONTRIBUTING.md#releasing)).
+To get started with development:
+
+- [Install Go](https://golang.org/doc/install).
+- Ensure you have `make` installed, as this project uses a Makefile.
+- Fork this repository and clone your fork locally.
+- Make your changes (see [Formatting](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/blob/master/CONTRIBUTING.md#formatting)) and commit to your fork. Use [Conventional Commits](https://www.conventionalcommits.org/) style for commit messages.
+- Add relevant unit tests (see [Testing](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin/blob/master/CONTRIBUTING.md#testing)) for your changes.
+- Update documentation if necessary.
+- Open a pull request; required checks will automatically run on your changes.
```

Comment:
This is not true ("will automatically run ..."). We have the block step that a maintainer will need to unblock to run the tests